Directory.Build.props

```diff
@@ -29,6 +29,12 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'">
```

Comment:
@westin-m - how are you meansuring the trimmed size? Do you just look at the size of the DLL ?

Reply:
Currently yes, I am looking at the size of the .dll files.